singletons.cabal

```diff
@@ -55,6 +55,7 @@ library
                       template-haskell,
                       containers >= 0.5,
                       th-desugar >= 1.9 && < 1.10,
+                      pretty,
```

Comment:
Note that this is shipped with GHC, so this doesn't actually add a new dependency _per se_.